### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by danintel**
* Add Consensus Answer
* Update Docker Proxy Question
* Add Application Development Video Link
* Add NEXT Directory Chip Chat Link

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

**Review by dcmiddle — APPROVED** [review]

File: generator/source/faq/consensus.rst [hunk 1]
```diff
@@ -219,6 +219,11 @@ Blockchain achieves fault tolerance by having its state (data) completely duplic
 Distributing nodes on virtual machines sharing the same host does nothing to guard against hardware faults.
 Distributing nodes at the same site does not protect against site outages.
 
+Can I restrict what validator nodes win consensus?
+-------------------------
+No. Every peer node validates blocks and every peer node can publish a block.
+One can write your own plugin consensus module to restrict what peer nodes win. Or modify an existing consensus module to experiment.
```
Comment (annechenette) [inline generator/source/faq/consensus.rst]: Minor nitpick:  I'd say either "You can write your own ...." or "One can write one's own...".

File: generator/source/faq/videos.rst [hunk 2]
```diff
@@ -31,6 +31,12 @@ Intel: Building Blockchain for the Enterprise. Shows how Sawtooth builds are don
 Sawtooth 1.0 Architecture and App Development (31:26, Zac Delventhal, Bitwise, 2018)
     https://www.youtube.com/watch?v=uBebFQM49Xk
 
+Hyperledger Sawtooth Technical Overview Development (58:50, Dan Anderson, Intel, 2018)
```
Comment (annechenette) [inline generator/source/faq/videos.rst]: Delete "Development" (the title in the video is "Hyperledger Sawtooth Technical Overview").

File: generator/source/faq/videos.rst [hunk 3]
```diff
@@ -31,6 +31,12 @@ Intel: Building Blockchain for the Enterprise. Shows how Sawtooth builds are don
...
+    https://www.youtube.com/watch?v=KGFuKIBma_M
+
+Hyperledger Sawtooth Application Development (55:16, Dan Anderson, Intel, 2018)
```
Comment (annechenette) [inline generator/source/faq/videos.rst]: This is the "text title" of the youtube video, but I prefer the title shown in the actual video: Hyperledger Sawtooth Application Developer Overview using "CookieJar" (it's more descriptive).

File: generator/source/faq/videos.rst [hunk 4]
```diff
@@ -61,6 +67,9 @@ Hyperledger Sawtooth 1.0: Market Significance and Technical Overview. Free regis
 
 Intel Chip Chat Audio
 ---------------------
+Revolutionizing Identity Management with Sawtooth Blockchain. Ep. 612 (8:33, Chris Spanton, T-Mobile, 2018)
```
Comment (annechenette) [inline generator/source/faq/videos.rst]: The title doesn't include "Sawtooth". It's: Revolutionizing Identity Management with Blockchain – Intel Chip Chat – Episode 612

**Review by annechenette — COMMENTED** [review]
A wording nitpick and comments on the link titles.

File: generator/source/faq/consensus.rst (hunk 1 above)
Comment (danintel) [reply generator/source/faq/consensus.rst]: OK. "You can ..."

File: generator/source/faq/videos.rst (hunk 2 above)
Comment (danintel) [reply generator/source/faq/videos.rst]: OK, thanks.

File: generator/source/faq/videos.rst (hunk 4 above)
Comment (danintel) [reply generator/source/faq/videos.rst]: I changed it to "[Sawtooth] Blockchain". It discusses Sawtooth, even though it is not in the title.

**Review by annechenette — APPROVED** [review]

**Pull request merged**